src/Dev/SapphireTest.php

```diff
@@ -327,6 +333,12 @@ protected function setUp()
         Email::config()->remove('bcc_all_emails_to');
     }
 
+    public static function setUpBeforeClass()
+    {
+        self::$test_singleton = singleton(static::class);
```

Comment:
It may be wise to put `static::start()` in here to ensure that the manifest is configured for tests prior to any code touching injector (as `singleton()` does).